src/offsetFunctions.ts

```diff
@@ -139,7 +138,7 @@ function offsetToPathsOrPolyTree(
       return offset.executeToPaths(params.delta, params.cleanDistance);
     } else {
       if (params.cleanDistance !== undefined) {
-        throw new ClipperError("cleaning is not available for poly tree results");
+        return undefined; // cleaning is not available for poly tree results
```

Comment:
why was this changed?

Reply:
It's eslint's "using exceptions for local control flow" lint.

The only thing that ever could happen to an exception thrown here is the catch block on lines 146-147, which tosses the error away and returns `undefined`, so it's the same thing here.